INSTALL.md

```diff
@@ -168,6 +151,18 @@ particular pod:
   $ fission function logs --name hello --pod <pod name>
 ```
 
+* (Optional) Edit `fission-logger.yaml` to add Influxdb username and password and following env to controller deployment.  Then apply `fission-logger.yaml` and new controller deployment.
```

Comment:
I suggest just removing this section. We should generate a random password in the helm chart (the plan is to move to that as the main recommended installation method).